src/lib/themes.js

```diff
@@ -12,32 +15,39 @@ export const bgImages = require.context(
 
 const defaultTheme = presetThemesContext("./default.json");
 
-export const colorToCSS = color => {
-  const { h, s, l, a } = color;
-  return typeof a === "undefined"
-    ? `hsl(${h}, ${s}%, ${l}%)`
-    : `hsla(${h}, ${s}%, ${l}%, ${a * 0.01})`;
+export const themesEqual = (themeA, themeB) =>
+  // HACK: "deep equal" via stringify
+  // http://www.mattzeunert.com/2016/01/28/javascript-deep-equal.html
+  JSON.stringify(themeA) === JSON.stringify(themeB);
+
+export const makeTinycolor = colorIn => {
+  let a = colorIn.a;
+  if (typeof a !== "undefined" && a > 1.0) {
```

Comment:
This is mainly for backwards-compatibility with themes already in localstorage with the old HSLA format. We could drop this eventually.